addons/mail/static/src/components/message/message.xml

```diff
@@ -49,7 +49,7 @@
                                 />
                             </t>
                         </div>
-                        <t t-if="message.isCurrentPartnerAuthor and threadViewer and threadViewer.thread">
+                        <t t-if="message.isCurrentPartnerAuthor and threadViewer and threadViewer.thread and !threadViewer.thread.mass_mailing">
```

Comment:
why no seen indicator for mailing channels?

Reply:
I think it was not in old code.
And there were shown on every message, so I supposed received data were somehow not good for that feature when mailing was involved. If that's not the case then there could be a problem with seen indicators

Reply:
If seen data are no good with mailing channels, and it's too complex to fix (likely so), then I would prefer having a thread field like `hasSeenIndicator`, which could be customized so that seen feature is disabled for a subset of threads.

Reply:
I think the problem was/is that we don't want to send seen notif for group channels unfortunately due to the perf issue, but the server is still giving us the seen status for some messages (especially those sent by email that are automatically marked as seen). So while this feature is disabled, we have to go and disable it the hard way otherwise we randomly get the indicator sometimes. I think the condition might even be more strict and limit to only those that are specifically allowed (chat and livechat only?).